bokeh/server/views/autoload_js_handler.py

```diff
@@ -8,6 +8,7 @@
 
 from six.moves.urllib.parse import urlparse
 from tornado import gen
+from tornado.web import MissingArgumentError
```

Comment:
I forgot to remove this line. We don't need it so I'll remove it in a subsequent PR.